src/crafting.cpp

```diff
@@ -201,14 +205,18 @@ void finalize_recipes()
     }
 }
 
-bool player::crafting_allowed()
+bool player::crafting_allowed(const std::string rec_name) {
+    return crafting_allowed(*recipe_dict[rec_name]);
+}
```

Comment:
@Coolthulhu thanks, I missed that. I've been trying to be better about that. Having fixed it... I hate one-character changes to player.h that require me to recompile the whole damn game :/